python/dask_cudf/dask_cudf/backends.py

```diff
@@ -218,6 +219,11 @@ def categorical_dtype_cudf(categories=None, ordered=None):
     return cudf.CategoricalDtype(categories=categories, ordered=ordered)
 
 
+@tolist_dispatch.register((cudf.Series, cudf.Index))
+def tolist_cudf(obj):
+    return obj.to_arrow().to_pylist()
```

Comment:
Since we're registering it like this here, should we change our exception messages and internal implementations to use `to_arrow().to_pylist()` instead of `to_pandas().to_list()`?

Reply:
This actually makes sense, changed it to have consistent behavior.